src/music.ts

```diff
@@ -110,18 +111,18 @@ export class Music {
     };
   }
 
-  static get scaleTypes() {
+  static get scaleTypes(): Record<string, number[]> {
     return {
       M: Music.scales.major,
       m: Music.scales.minor,
     };
   }
 
-  static get accidentals() {
+  static get accidentals(): string[] {
     return ['bb', 'b', 'n', '#', '##'];
   }
 
-  static get noteValues() {
+  static get noteValues(): Record<string, IMusicNoteValue> {
     return {
       c: { root_index: 0, int_val: 0 },
       cn: { root_index: 0, int_val: 0 },
```

Comment:
Since it is Record<string, ...>, I would also put also all the keys in quotes, not only for consistency sake, but also for better readability. BTW keys in javascript object are always of string type

Reply:
@mscuthbert Mike, @0xfe asked us to include you in the review.

I hope you like it, we are already preparing the next file proposed: tunning.ts
Do you have other proposals? I would say 5 classes would be something reasonable for the next batch.

Reply:
@0xe what is your oppinion on this one

Reply:
Personally I agree with @wassertim. We did not  make the change because this implies changing a rule in prettier which with the current configuration automatically changes other files not related to this PR. We decided to wait here for a general concensus.

Reply:
I suggest to leave it as it is on the first PR. We will see this topic again and if we change the configuration, it will resolved automatically.